processing/src/main/java/io/druid/query/DefaultGenericQueryMetricsFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import io.druid.jackson.DefaultObjectMapper;
+
+public class DefaultGenericQueryMetricsFactory implements GenericQueryMetricsFactory
+{
+  private static final DefaultGenericQueryMetricsFactory INSTANCE = new DefaultGenericQueryMetricsFactory(new DefaultObjectMapper());
```

Comment:
@nishantmonu51 I think I inject ObjectMapper a few lines below. This static final instance is used only in tests, directly or indirectly: by filling QueryMetricsFactory constructor parameters of some QueryToolChests; those shorter constructors of QueryToolChests are used in tests.

Or should I inject it somehow in static final?